readme.md

```diff
@@ -238,7 +259,7 @@ const limiter = rateLimit({
 
 The response body to send back when a client is rate limited.
 
-May be a `string`, JSON object, or any other value that Express's
+May be a `string`, JSON object, function, or any other value that Express's
 [response.send](https://expressjs.com/en/4x/api.html#response.send) method
 supports.
```

Comment:
```suggestion
May be a `string`, JSON object, or any other value that Express's
[`response.send`](https://expressjs.com/en/4x/api.html#res.send) method
supports. It can also be a (sync/async) function that accepts the Express
request and response objects and then returns a `string`, JSON object or
any other value the Express `response.send` function accepts.
```

Also, could you please change the `any` above this line to `any | function`?